README.rst

```diff
@@ -4,7 +4,19 @@
 Zulip for Android
 =================
 
-This is the Android client for `Zulip <http://www.zulip.org>`_.  It is
+**Note**: We've decided that our `React Native app`_ is far enough along that it
+makes sense to focus our efforts on making the React Native app better on Android
+and iOS rather than continuing to develop two apps in parallel. Hence, for GSoC
+we will be focused on making the React Native app awesome. The plan is in
+this `document`_, which also covers important FAQ for anyone interested in doing
+GSoC on Zulip mobile apps.
+
+.. _React Native app: https://github.com/zulip/zulip-mobile
+.. _document: https://paper.dropbox.com/doc/Zulip-Android-plan-draft-1ucRhuH9LMNooR7WMsJ37
+
+---------------
+
+This is the native Android client for `Zulip <http://www.zulip.org>`_.  It is
```

Comment:
Let's move the paper doc into docs/android-strategy.md in this repository and link to it from here, since it's kinda not ideal to require folks to use Dropbox paper to read it (won't be googleable, etc.).

Otherwise this message looks pretty good.